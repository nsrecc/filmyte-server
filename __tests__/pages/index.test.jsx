import React from 'react';
import { shallow } from 'enzyme';
import Home from 'pages/index';

describe('Home', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
