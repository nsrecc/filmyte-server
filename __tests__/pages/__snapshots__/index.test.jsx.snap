// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches snapshot 1`] = `
<div
  className="container"
>
  <Head>
    <title>
      Filmyte
    </title>
    <link
      href="/favicon.ico"
      rel="icon"
    />
  </Head>
  <main
    className="main"
  >
    <h1
      className="title"
    >
      Welcome to Filmyte web server!
    </h1>
    <h2>
      Visit the 
      <Link
        href="/api/graphql"
      >
        Filmyte web server GraphQL playground
      </Link>
      .
    </h2>
    <h2>
      Visit the
       
      <Link>
        Filmyte web client
      </Link>
      .
    </h2>
  </main>
</div>
`;
